T2D-207 updated fb server context

selvaebi opened the pull request.

selvaebi committed:
updated fb server context

Zhicheng-Liu commented on Jenkinsfile:
```diff
@@ -57,7 +57,7 @@ pipeline {
       steps {
         echo 'Deploying to Fallback'
-        sh "curl --upload-file fallback/target/amp-t2d-property-registry-*.war 'http://'${tomcatCredentials}'@'${fallbackHost}':8080/manager/text/deploy?path=/ega/ampt2d/registry&update=true' | grep 'OK - Deployed application at context path '"
+        sh "curl --upload-file fallback/target/amp-t2d-property-registry-*.war 'http://'${tomcatCredentials}'@'${fallbackHost}':8080/manager/text/deploy?path=/ega/t2d/registry&update=true' | grep 'OK - Deployed application at context path '"
```
For Staging deployment the context is using "ampt2d" while on productions it is "t2d". Could you make them consistent?

selvaebi replied:
mm i am just testing around , its kinda tricky , if i use ampt2d it is directly using system's mapping, if i dont use ampt2d swagger is fine but direct json hrefs are broken :(

selvaebi committed:
made t2d instead of ampt2d 

made t2d instead of ampt2d to access ursl through zuul proxy instead of system's mapping directly

selvaebi committed:
corrected url for proxy
